? cr17.patch
? hibernate.cfg.bin
? test/org/hibernate/test/legacy/ABC.hbm.xml.bin
? test/org/hibernate/test/legacy/ABCExtends.hbm.xml.bin
? test/org/hibernate/test/legacy/Baz.hbm.xml.bin
? test/org/hibernate/test/legacy/Blobber.hbm.xml.bin
? test/org/hibernate/test/legacy/Broken.hbm.xml.bin
? test/org/hibernate/test/legacy/Category.hbm.xml.bin
? test/org/hibernate/test/legacy/Circular.hbm.xml.bin
? test/org/hibernate/test/legacy/Commento.hbm.xml.bin
? test/org/hibernate/test/legacy/ComponentNotNullMaster.hbm.xml.bin
? test/org/hibernate/test/legacy/Componentizable.hbm.xml.bin
? test/org/hibernate/test/legacy/Container.hbm.xml.bin
? test/org/hibernate/test/legacy/Custom.hbm.xml.bin
? test/org/hibernate/test/legacy/CustomSQL.hbm.xml.bin
? test/org/hibernate/test/legacy/Eye.hbm.xml.bin
? test/org/hibernate/test/legacy/Fee.hbm.xml.bin
? test/org/hibernate/test/legacy/Fo.hbm.xml.bin
? test/org/hibernate/test/legacy/FooBar.hbm.xml.bin
? test/org/hibernate/test/legacy/Fum.hbm.xml.bin
? test/org/hibernate/test/legacy/Fumm.hbm.xml.bin
? test/org/hibernate/test/legacy/Glarch.hbm.xml.bin
? test/org/hibernate/test/legacy/Holder.hbm.xml.bin
? test/org/hibernate/test/legacy/IJ2.hbm.xml.bin
? test/org/hibernate/test/legacy/Immutable.hbm.xml.bin
? test/org/hibernate/test/legacy/Location.hbm.xml.bin
? test/org/hibernate/test/legacy/Many.hbm.xml.bin
? test/org/hibernate/test/legacy/Map.hbm.xml.bin
? test/org/hibernate/test/legacy/Marelo.hbm.xml.bin
? test/org/hibernate/test/legacy/MasterDetail.hbm.xml.bin
? test/org/hibernate/test/legacy/Middle.hbm.xml.bin
? test/org/hibernate/test/legacy/Multi.hbm.xml.bin
? test/org/hibernate/test/legacy/MultiExtends.hbm.xml.bin
? test/org/hibernate/test/legacy/Nameable.hbm.xml.bin
? test/org/hibernate/test/legacy/One.hbm.xml.bin
? test/org/hibernate/test/legacy/ParentChild.hbm.xml.bin
? test/org/hibernate/test/legacy/Qux.hbm.xml.bin
? test/org/hibernate/test/legacy/Simple.hbm.xml.bin
? test/org/hibernate/test/legacy/SingleSeveral.hbm.xml.bin
? test/org/hibernate/test/legacy/Stuff.hbm.xml.bin
? test/org/hibernate/test/legacy/UpDown.hbm.xml.bin
? test/org/hibernate/test/legacy/Vetoer.hbm.xml.bin
? test/org/hibernate/test/legacy/WZ.hbm.xml.bin
Index: src/org/hibernate/type/EntityType.java
===================================================================
RCS file: /cvsroot/hibernate/Hibernate3/src/org/hibernate/type/EntityType.java,v
retrieving revision 1.49
diff -r1.49 EntityType.java
371a372,378
> 
>         Serializable id = null;
>         if (x instanceof HibernateProxy) {
>             id = ( (HibernateProxy) x ).getHibernateLazyInitializer().getIdentifier();
>         } else {
>             id = persister.getIdentifier(x, entityMode);
>         }
373,377d379
< 		if (x instanceof HibernateProxy) {
< 			return ( (HibernateProxy) x ).getHibernateLazyInitializer().getIdentifier().hashCode();
< 		}
< 		
< 		Serializable id = persister.getIdentifier(x, entityMode);
